src/raps/swapAndDepositCompound.js

```diff
@@ -24,6 +24,15 @@ export const estimateSwapAndDepositCompound = async ({
   outputReserve,
 }) => {
   const { accountAddress, chainId, network } = store.getState().settings;
+  const { pairs, allPairs } = store.getState().uniswap;
+  const globalPairs = {
+    ...pairs,
+    ...allPairs,
```

Comment:
Shouldn't the order be the opposite? assuming we want to preserve the overrides

Reply:
this is not user facing so I guess it doesn't matter

Reply:
Yeah I think once it gets to user-facing, we do an extra pass on the token overrides list anyways, but this will be tweaked with the V2 upgrade anyway!